src/docs/self-hosted/index.mdx

```diff
@@ -107,6 +107,31 @@ By default Sentry sends anonymous usage statistics to the Sentry team. It helps
 
 You can disable this feature by adding `SENTRY_BEACON = False` into `sentry.conf.py` file.
 
+### Errors-only Self-hosted Sentry
+
+This feature is experimental and now in beta. Starting from 24.8.0+ onwards, users will have the ability to choose between two distinct types of self-hosted Sentry deployments, tailored to their specific needs and preferences.
+
+**Errors Only**\n
+Opt for a sleek and efficient version of self-hosted Sentry that focuses on what we do best—tackling issues head-on. This lightweight option includes these features:
+1. Issues
+2. Alerts
+3. Integrations
+4. Dashboards
+5. Releases
+6. Discover
+
+**Feature Complete**\n
+This is our default version of self-hosted Sentry that is most similar feature parity wise to our SaaS product. This includes everything offered in the Errors Only version, as well as many other amazing features.
+1. Traces
+2. Profiles
+3. Replays
+4. Insights (Requests, Queries, Assets, etc)
+5. User Feedback
+6. Performance
+7. Crons
+
+In order to enable errors only self-hosted, you'll need to update your [.env file](https://github.com/getsentry/self-hosted/blob/master/.env) to include `COMPOSE_PROFILES=errors-only`.
```

Comment:
This should go in the `Errors Only` section above. This section should read something like:

"To use Feature Complete Sentry, you'll need to ensure that you are not setting the `COMPOSE_PROFILES` environment variable (usually set via your `.env` file) when running Sentry via docker compose."